test/Sema/keypath_subscript_nolabel.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: %target-swift-frontend -typecheck -verify -primary-file %s
+// [SR-12745]
+// rdar://problem/62957095
+struct S {
+  var x : Int = 0
+}
+var s = S()
```

Comment:
@artemcm 

Do you think it is worth adding tests where there are other subscripts declared that doesn't match 

```swift
struct S {
   var x : Int = 0
   subscript(_ v: Int) -> Int { 0 }  
 }
 var s = S()
 s[\.x] = 10 // ... 
```